web-server/app/org/genivi/webserver/controllers/Application.scala

```diff
@@ -76,7 +80,12 @@ class Application @Inject() (ws: WSClient, val messagesApi: MessagesApi, ldapAut
     wreq.execute.map { resp =>
       Result(
         header = ResponseHeader(resp.status, resp.allHeaders.mapValues(x => x.head)),
-        body = Enumerator(resp.bodyAsBytes))
+        body = {
+          val bstr = resp.bodyAsBytes
+          val blen = bstr.length.toLong
+          play.api.http.HttpEntity.Streamed(Source.single(bstr), contentLength = Some(blen), contentType = None)
```

Comment:
Isn't `resp.bodyAsBytes` a `Source`? It's strange to wrap some bytes in a `Source.single`, `asBytes` should be some kind of source. is there some conversion from a request body to a Source[Byte]  ?